### ISSUE TITLE
Formatting the contents to Look Better and Easier to Find

### ISSUE THREAD

**Issue by cybergeekgyan** [himahuja/Research-Internships-for-Undergraduates#85]
If we can follow this formatting for easier navigation and Application Last Dates to be in notice, If you like this I would love to do this for the whole repository.

|  Research opportunity | Eligibility Criteria | Location(Country) | Time required(in months) | Application Opening Date/Month | Application Closing Date | 
|-----------------------|-----------------|----------------------|-----------------|----------------------|-----------------|

**Comment by Neptune2003** [comment]
Ayo, its fine. It varies every year anyway

**Comment by himahuja** [comment]
I have been interested in this for a while, if you're willing to work on it. I have one concern though. On one end, what you suggest does make things more accessible, but on the other it might make it harder for the community to edit. Let's keep discussing the best formatting before you work on this. I'll turn my notifications on this. 

Also, what do you mean by the 'time required' column?

**Comment by ianitow** [comment]
> 

I don't think this would be harder for the community and even if it was, the benefits are more than the disadvantages. And the table format just giving an example I think will be enough because most of us that are using GitHub are developers used to doing this and most probably will be who contribute. (not sure)

**Comment by himahuja** [comment]
I am down if you wanna take this formatting over @cybergeekgyan

**Comment by cybergeekgyan** [comment]
Yes sure I would love to work on this and Keep it updated @himahuja
